provisioning/provisioning-service-client/src/main/java/com/microsoft/azure/sdk/iot/provisioning/service/configs/AllocationPolicy.java

```diff
@@ -3,15 +3,24 @@
 
 package com.microsoft.azure.sdk.iot.provisioning.service.configs;
 
+import com.google.gson.annotations.SerializedName;
+
 /**
  * The Device Provisioning Service enrollment level allocation policies.
  *
  * @see <a href="https://docs.microsoft.com/en-us/azure/iot-dps/tutorial-provision-multiple-hubs#set-the-device-provisioning-service-allocation-policy">
  *     Provision devices across load-balanced IoT hubs</a>
  */
 public enum AllocationPolicy {
+    @SerializedName("hashed")
```

Comment:
a unit test for this?

Reply:
Sure thing. I'll add a unit test for each of these possible values in both the individual and group enrollment unit tests